src/main/java/com/box/sdk/internal/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * <b>Not a part of Public API - the backward-compatibility is not maintained.</b><br />
+ * Whole 'internal' package is not a part of the public API and package (and all sub-packages and their classes) are determined only for SDK
+ * implementation.<br />
+ */
+package com.box.sdk.internal;
```

Comment:
Yes, it is, but it is conventional to have multiple packages within Java. Not to have 71 classes within single package.
Do you have any concrete requirement for single package solution?